schemas/dll.yml

```diff
@@ -0,0 +1,31 @@
+---
+- name: dll
+  title: DLL
+  group: 2
+  short: These fields contain information about loaded process modules.
+  description: >
+    These fields contain information about loaded process modules.
+
+    Many operating systems call process modules different names, but this field set refers all of the following:
+    * Dynamic-link library (`.dll) commonly used in Windows
```

Comment:
i'm not attached to the "commonly used in" phrase and am open to suggestions for the description